modules/virt-enabling-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-usb-host-passthrough_{context}"]
+= Enabling USB host passthrough
+
+You can enable USB host passthrough at the cluster level.
+
+.Prerequisites
+
+* Enabling the passthrough requires cluster administrator privileges.
+
+.Procedure
+
+. Identify the USB device vendor and product by running the following command:
++
+[source,terminal]
+----
+$ lsusb
+----
+
+. Open the HyperConverged (HCO) custom resource (CR) by running the following commmand:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Edit the CR by performing the following actions, as shown in the example that follows:
+
+.. Enable the `HostDevices` feature gate by adding it to the `spec.featureGates` stanza.
+.. Add a USB device to the `permittedHostDevices` stanza.
+
++
+.Example YAML snippet
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+   name: kubevirt-hyperconverged
+   namespace: {CNVNamespace}
+spec:
+  featureGates:
+    hostDevices: True <1>
+# ...
+  configuration:
+    permittedHostDevices: <2>
+      usb: <3>
+        - resourceName: kubevirt.io/peripherals <4>
+          selectors:
+            - vendor: "045e"
+              product: "07a5"
+            - vendor: "062a"
+              product: "4102"
+            - vendor: "072f"
+              product: "b100"
+
+----
+<1> Enables the `hostDevices' feature gate.
+<2> Lists the host devices that have permission to be used in the cluster.
+<3> Lists the available USB devices.
+<4> Uses `resourceName: deviceName` for each device added and assigned to the VM.
```

Comment:
not sure if 100% clear but the above would mean that the single resource name `kubevirt.io/peripherals` is binded to 3 usb devices at the same time. That means that when the user allocate one `kubevirt.io/peripherals`, the VM is only scheduled on a Node with this 3 USB devices connected at the same time in which all 3 are going to be redirected to the VM.

In case of doubt, the following commit log has some examples:
https://github.com/kubevirt/kubevirt/commit/47d4534dedae49cfcbe58f6367ea196c5dc12885

Reply:
Hi @victortoso , and thanks for looking! Are you saying that having the VM scheduled on a node with three USB devices is not how the USB passthrough should be configured? Or are you saying that it would be helpful to explain this further in the documentation so that it is more clear? 

I did see this in the link you provided:

A few helpful assumptions that make understanding the whole:

 1. We create one USB Device Plugin per resource name
 2. Each USB Selector defines a single USB device
 3. **Each resource name can have multiple USB selectors**

Reply:
My comment is just to be sure that the example and the expectation are in sync. If they are, we are good.
The idea behind multiple usb devices per k8s device is to allow multiple use cases like usb + keyboard or devices that might depend on each other.